zip-0302.rst

```diff
@@ -45,13 +45,27 @@ contents of a memo. It does not define consensus requirements.
      + strip any trailing zero bytes
      + decode it as a UTF-8 string (if decoding fails then report an error).
 
++ If the first byte has a value of 0xF5, then the reader MUST:
+     + Interpret the next few bytes (1 to 9 of them) as a 64-bit unsigned variable-length
+       integer [#Bitcoin-CompactSize]_, and use it as an arbitrary application-defined
+       "type" field.
```

Comment:
Calling this "arbitrary" makes it hard to have a common list of types. We should instead take the approach of ZIP 316 where we have a well-defined list of types, that can be amended.